### PR TITLE
Fix RingRange string method

### LINKED ISSUE #383: Fix RingRange string method
On this line:
https://github.com/k8ssandra/management-api-for-apache-cassandra/blob/feature/client-gen/management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java#L132
The logic incorrectly chops 2 characters off the end of the string. It should only chop 1.

### PR DESCRIPTION
**Pull request by emerkle826**
Fixes #383

### CODE REVIEW

File: management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java [hunk 1]
```diff
@@ -122,14 +123,15 @@ private String getParallelismName(RepairParallelism parallelism) {
...
+  @VisibleForTesting
+  String getRingRangeString(List<RingRange> associatedTokens) {
     if (associatedTokens != null && !associatedTokens.isEmpty()) {
       StringBuilder sb = new StringBuilder();
       for (RingRange ringRange : associatedTokens) {
         sb.append(toRangeString(ringRange)).append(",");
       }
       // remove trailing comma
-      return sb.substring(0, sb.length() - 2);
+      return sb.substring(0, sb.length() - 1);
```
Comment (Miles-Garnsey) [inline management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java:134]: Suggestion: for some reason I'm more comfortable with something that looks like this:

```
return associatedTokens.stream().map(i -> toRangeString(i)).collect(Collectors.joining(","));
```

This is stylistic, but I feel like it avoids a two step process and some of the off-by-one issues we've just seen. Up to you.

**Review by Miles-Garnsey — CHANGES_REQUESTED** [review]
A few suggestions, mostly looks good to me. Thanks for catching this!

File: management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java [hunk 2]
```diff
@@ -122,14 +123,15 @@ private String getParallelismName(RepairParallelism parallelism) {
     return parallelism != null ? parallelism.getName() : null;
   }
 
-  private String getRingRangeString(List<RingRange> associatedTokens) {
+  @VisibleForTesting
+  String getRingRangeString(List<RingRange> associatedTokens) {
```
[the remark this answers is not in the document]
Comment (emerkle826) [reply management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java:127]: For the record, I'm pretty sure I created this method in a mini-refactor, so this one is on me.

File: management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java (hunk 1 above)
Comment (emerkle826) [reply management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java:134]: will do

File: management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Miles-Garnsey) [reply management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java:128]: I know ;) But I probably still approved it!

**Review by Miles-Garnsey — APPROVED** [review]

**Pull request merged**